app/_hub/kong-inc/mtls-auth/index.md

```diff
@@ -29,44 +29,78 @@ params:
+        `DN` (Distinguished Name). Once `skip_consumer_lookup` is applied, any client with a
+        valid certificate can access the Service/API. To restrict usage to only some of the authenticated users,
+        also add the ACL plugin (not covered here) and create allowed or denied groups of users.
     - name: revocation_check_mode
       default: "`IGNORE_CA_ERROR`"
+      required: false
+      datatype: string
       description: |
-        >**Known Issue:** The default value `IGNORE_CA_ERROR` has a known issue in versions 1.5.0.0 and later. As a workaround, manually set the value to `SKIP`.
-
-        Controls client certificate revocation check behavior. Valid values are `SKIP`, `IGNORE_CA_ERROR`, or `STRICT`. If set to `SKIP`, no revocation check will be performed. If set to `IGNORE_CA_ERROR`, the plugin will respect the revocation status when either OCSP or CRL URL is set, and will not fail on network issues. If set to `STRICT`, the plugin will only treat the certificate as valid when it's able to verify the revocation status, and a missing OCSP or CRL URL in the certificate or a failure to connect to the server will result in a revoked status. If both OCSP and CRL URL are set, the plugin always checks OCSP first, and will only check CRL URL if it can't communicate with the OCSP server.
+        >**Known Issue:** The default value `IGNORE_CA_ERROR` has a known issue in versions 1.5.0.0 and later.
+        As a workaround, manually set the value to `SKIP`.
+
+        Controls client certificate revocation check behavior. Valid values are `SKIP`, `IGNORE_CA_ERROR`, or `STRICT`.
+        If set to `SKIP`, no revocation check will be performed. If set to `IGNORE_CA_ERROR`, the plugin will respect
+        the revocation status when either OCSP or CRL URL is set, and will not fail on network issues. If set to `STRICT`,
+        the plugin will only treat the certificate as valid when it's able to verify the revocation status, and a missing
+        OCSP or CRL URL in the certificate or a failure to connect to the server will result in a revoked status.
+        If both OCSP and CRL URL are set, the plugin always checks OCSP first, and will only check the CRL URL if
+        it can't communicate with the OCSP server.
     - name: http_timeout
       default: "30000"
+      datatype: number
       description: |
-        HTTP timeout threshold, in milliseconds, when communicating with the OCSP server or downloading CRL.
+        HTTP timeout threshold in milliseconds when communicating with the OCSP server or downloading CRL.
     - name: cert_cache_ttl
       default: "60000"
+      datatype: number
       description: |
-        The length of time, in milliseconds, between refreshes of the revocation check status cache.
+        The length of time in milliseconds between refreshes of the revocation check status cache.
+    - name: cache_ttl
```

Comment:
@murillopaula this cache_ttl has been missing for a few years. Added and used LDAP's description. Please verify.

Reply:
Very nice catch here. Verified.